src/content/selector.js

```diff
@@ -37,6 +37,16 @@ window['pctSelector'] = (function(window) {
         return selected;
     }
 
+    function isPostPreview() {
+        return (document.location.hash === '#newPost' || document.location.pathname.indexOf('createNewPost') >= 0);
```

Comment:
```suggestion
        return (document.location.hash == '#newPost' ||
                document.location.pathname.indexOf('createNewPost') >= 0);
```

Personal preference; I don't use === unless I actually care about type (ditto for line 45).

Spacing is personal preference, too. Generally* I stick to the good old 80(ish) character line, and break at reasonable points.

*There are plenty of exceptions to this, particularly in this code base.